soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series

```diff
@@ -16,6 +16,9 @@ config ITE_IT8XXX2_INTC
 config SYS_CLOCK_HW_CYCLES_PER_SEC
 	default 660000
 
+config COMPRESSED_ISA
```

Comment:
Would it be better if `SOC_SERIES_RISCV32_IT8XXX2` did an `imply` or `select` on `COMPRESSED_ISA`? Afaict, this option is already avalable for all RiscV.

Reply:
Done